docs/containers-registries.conf.5.md

```diff
@@ -82,7 +82,7 @@ location = "internal-registry-for-example.net/bar"
 requests for the image `example.com/foo/myimage:latest` will actually work with the
 `internal-registry-for-example.net/bar/myimage:latest` image.
 
-With a `prefix` containing a wildcard in the format: "*.example.com" for subdomain matching,
+With any valid `prefix` with or without a wildcard in the format: "*.example.com" for subdomain matching,
 the location can be empty. In such a case,
```

Comment:
This sentence seems a bit too complicated. I suggest: "The `location` can be empty if the `prefix` is set."